docs/idea-setup.md

```diff
@@ -23,3 +24,14 @@ For each of the available services:
     4. EnablerV1SampleApplication - onboarding-enabler-spring-v1-sample-app
     5. GatewayApplication - gateway-service
 4. Run the service
+
+#### If using IDEA Community Edition
+
+For each of the available services:
+
+1. Run -> Edit configurations...
+2. Create New (Ctrl + N or Cmd + N) -> Application
+3. Choose main class (E.g `org.zowe.apiml.gateway.GatewayApplication`)
+4. Add Environment varible `spring.config.additional-location` and it's value `file:./config/local/{SERVICE_NAME}.yml` Replace SERVICE_NAME with respective main class (E.g `file:./config/local/gateway-service.yml`)
```

Comment:
this should be done for all services you want to run

Reply:
there are specific local config overrides for discovery, gw, catalog and discoverable client. I suggest you include them as well